src/main/java/net/coderbot/iris/gl/uniform/Uniform.java

```diff
@@ -3,7 +3,7 @@
 public abstract class Uniform {
 	protected final int location;
 
-	Uniform(int location) {
+	public Uniform(int location) {
```

Comment:
Why does this need to be public?

Reply:
Oh, this is shouldn't be in this commit. I think in one of my latest branches this doesn't even need to be public anymore. 

I'll remove it.